pwa-shell/src/ui/notifications/NotificationsHeader.js

```diff
@@ -1,29 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl.macro';
 
-const text = amount => {
-  switch(amount) {
-    case 0: return 'Non ci sono nuove notifiche';
-    case 1: return (
-      <React.Fragment>
-        Hai <span className="NotificationsHeader__amount">1</span> nuova notifica
-      </React.Fragment>
-    );
-    default: return (
-      <React.Fragment>
-        Hai <span className="NotificationsHeader__amount">{amount}</span> nuove notifiche
-      </React.Fragment>
-    );
-  }
-};
-
-const NotificationHeader = ({ notificationAmount }) => (
-  <div className="NotificationsHeader px-3 pb-3">
-    <div className="NotificationsHeader__title">Notifiche</div>
-    <div className="pt-1">{ text(notificationAmount) }</div>    
-  </div>
+const text = (amount) => (
```

Comment:
You're going in the right direction, well done :)
But at his point you can get rid of the `React.Fragment` tag (no more useful, as it as one child only) as well as the whole `text` method, putting the remaining markup straight into `NotificationHeader`